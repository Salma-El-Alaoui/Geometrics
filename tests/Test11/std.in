C c1 1 1 1
MOVE c1 1 1
LIST
MOVE c1 -1 -1
LIST
R r1 1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
LIST
MOVE r1 1 1
MOVE l1 1 1
MOVE pl1 1 1
LIST
MOVE aaa 1 1
MOVE c1 1 1 1
MOVE c1 1 a
EXIT
