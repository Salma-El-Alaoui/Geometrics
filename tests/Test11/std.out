OK
#Element ajoute: c1
OK
#c1 deplace
C c1 2 2 1
OK
#c1 deplace
C c1 1 1 1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
OK
#Element ajoute: pl1
C c1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
R r1 1 1 1 1
OK
#r1 deplace
OK
#l1 deplace
OK
#pl1 deplace
C c1 1 1 1
L l1 2 2 2 2
PL pl1 2 2 2 2 2 2 2 2
R r1 2 2 2 2
ERR
#L'element aaa n'existe pas 
ERR
#Parametres invalides
ERR
#Parametres invalides
